### ISSUE TITLE
Manage contact ACL issue

### ISSUE THREAD

**Issue by xsylx** [shinken-monitoring/mod-webui#630]
Shinken Version : 2.4.2
Webui2 Version : 2.7.2

Hello Shinken addicts,

I'am looking for the possibility to enable actions for my Shinken users (non-admin). When i read the doc, we have to set "manage_acl" to "0" on the webui2.cfg and "can_submit_commands = 1" on the contact or contact template definition.

I set up this settings but my non-admin users cannot send actions on their host or services. (configuration was reloaded...)

# The settings on Webui2.cfg

![2018-11-15_10h38_16](https://user-images.githubusercontent.com/16323410/48544367-bb185180-e8c3-11e8-9769-77fda3f24fa0.png)

# The contact template definition 

![2018-11-15_10h41_11](https://user-images.githubusercontent.com/16323410/48544402-cd928b00-e8c3-11e8-9bb5-387fd2e0863f.png)

Can you help me ? Perhaps i missed up something....

**Comment by mohierf** [comment]
Indeed, set ting`manage_acl=0` is enough to allow all users to use commands from the Web UI ... what doi you mean exactly : they cannot send actions ? No action buttons ? Actions are not considered ?

**Comment by xsylx** [comment]
@mohierf Thanks for your answer. The users have no actions buttons on their Webui.

**Comment by maethor** [comment]
@SylvainGuibert is it still relevant or can I close this issue?
